### PR TITLE
Docs: Add .babelrc to CLI Docs

### LINKED ISSUE #1222: Cucumber + Babel = Syntax Error
Hey there,

I've tried to use cucumber with babel 7 but were not successful. 
The command I've used (based from the docs and google) requires the module @babel/register but that seem to have not the effect I intended.
```
./node_modules/.bin/cucumber-js --require-module="@babel/register"  src/test/bdd
```

My application is for sure written in modern javascript and my tests as well. Like described in the docs I'm importing After, Before, Given etc from the cucumber package. Running it causes this exception:

```
import { After, Before, Given } from 'cucumber';
       ^

SyntaxError: Unexpected token {
    at new Script (vm.js:84:7)
    at createScript (vm.js:264:10)
    at Object.runInThisContext (vm.js:312:10)
    at Module._compile (internal/modules/cjs/loader.js:694:28)
```

My cucumber version is 5.1.0.

Can you tell me what is the proper setup with cucumber and babel? The docs are not very clear about that.

### PR DESCRIPTION
**Pull request by Izhaki**
Fixes #1222

### CODE REVIEW

**Comment by charlierudolph** [comment]
Hey @Izhaki can you explain why cucumber requires node as a babel target? It makes sense but just curious if we did any digging as to what having a node target does?

File: docs/cli.md [hunk 1]
```diff
@@ -130,6 +130,20 @@ For instance, for ES6 support with [Babel](https://babeljs.io/) 7 add:
 
 This will effectivally call `require('@babel/register')` prior to requiring any support files.
 
+Note: Your `.babelrc` should target a node environment. For example:
+
+```json
+{
+  "presets": [
+    [ "@babel/preset-env", {
+      "targets": {
```
Comment (charlierudolph) [inline docs/cli.md]: I'd prefer "targets": "maintained node versions".

File: docs/cli.md [hunk 2]
```diff
@@ -130,6 +130,20 @@ For instance, for ES6 support with [Babel](https://babeljs.io/) 7 add:
 
 This will effectivally call `require('@babel/register')` prior to requiring any support files.
 
+Note: Your `.babelrc` should target a node environment. For example:
```
Comment (charlierudolph) [inline docs/cli.md]: This is only valid if you are using babel/preset-env. Think that is worth calling out

**Comment by Izhaki** [comment]
@charlierudolph 

All the comments makes sense. But I've made this change with a single objective:

**Prevent issues like #1222 where one is unaware of the need for `.babelrc` (and that it should target node).**

I doubt there are many people who's first dip into ES6 and babel on node will be through writing step definitions (there must be some code being tested right)?

So the whole point was to deliver it all in the most concise way, and explicitly make it ultra-short so to encourage people to go and read more in places that can elaborate on these topics. Basically: The bare minimum to get people off the ground.

Let's merge?

**Comment by charlierudolph** [comment]
I'm fine not mentioning babel preset-env as they would get an error when using that but I would prefer switching from the example specifically targeting node 4 to targeting maintained node versions as that will age better and that is what cucumber js supports.

Overall, I'm not sure this needs to be documented here. Given that its specific to babel on node, its really more of babel documentation. I'm also not sure this is a common enough question for us to put on our docs. This was just a single user still learning how to use bazel with node and that isn't really something I think we need to document.

**Comment by Izhaki** [comment]
I'm with you. Closing.

**Pull request closed without merging**